controllers/storagecluster/storagecluster_controller_test.go

```diff
@@ -1161,13 +1161,17 @@ func createFakeStorageClusterReconciler(t *testing.T, obj ...runtime.Object) Sto
 		operatorNamespace = "openshift-storage"
 	}
 
+	availCrds, err := statusutil.MapCRDAvailability(context.Background(), client, statusutil.GetCrds()...)
+	assert.NoError(t, err)
```

Comment:
A test should assume the existence/non-existence of specific CRs. Using the same util in a test code that depends on the util means that if the util is  broken the test might still succeed

Reply:
From the main, we are passing a new map of availableCRDs if we don't use this util function the map will always be empty and will result in exit of reconcile from os.Exit condition